frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx

```diff
@@ -175,8 +175,8 @@ export function PolicyAutomationWizard(props: {
                         hidden={(item) => !item.spec?.automationDef?.name}
                     />
                     <WizNumberInput
-                        path="spec.automationDef.policyViolationContextLimit"
-                        label={t('Policy Violation Context Limit')}
+                        path="spec.automationDef.policyViolationsLimit"
+                        label={t('Policy Violations Limit')}
```

Comment:
@mprahl Are you OK if other languages have the wrong string until 2.7.1?

Reply:
I'm okay with that. Thanks for pointing that out. I just updated the translation keys to match the new wording.